editor.planx.uk/src/pages/FlowEditor/components/Flow/components/Checklist.tsx

```diff
@@ -24,8 +25,7 @@ type Props = {
   type: TYPES;
   [key: string]: any;
   wasVisited?: boolean;
-} & NodeTags &
-  TemplatedNodeData;
```

Comment:
`props.data` _already_ captures tags and templated node data fields - we don't need to define them _again_ at the outer level here !